logstash-core/src/main/java/org/logstash/ConvertedMap.java

```diff
@@ -61,7 +61,7 @@ public static ConvertedMap newFromMap(Map<? extends Serializable, Object> o) {
     }
 
     public static ConvertedMap newFromRubyHash(final RubyHash o) {
-        return newFromRubyHash(WorkerLoop.THREAD_CONTEXT.get(), o);
+        return newFromRubyHash(o.getRuntime().getCurrentContext(), o);
```

Comment:
same comment; why not stay consistent by using `RubyUtil.RUBY.getCurrentContext()` ?

Reply:
thanks, seems fine for LS ... no reason to not use `RUBY.getCurrentContext()`, will update